onnx/defs/generator/defs.cc

```diff
@@ -110,27 +77,22 @@ ONNX_OPERATOR_SET_SCHEMA(
              "tensor(uint32)",
              "tensor(uint64)",
              "tensor(bool)"},
-            "Constrain output types. Strings and complex are not supported.")
+            "Constrain output types")
```

Comment:
Do we need to constrain output types?

Reply:
Since we have the same concern in ConstantLike... I am ok with removing this constrain.